presto-main/src/test/java/io/prestosql/server/protocol/TestQueryResultRows.java

```diff
@@ -486,7 +486,7 @@ public void accept(Throwable throwable)
         }
     }
 
-    private static Type createMapType(Type keyType, Type valueType)
```

Comment:
instead of removing, use the `valueType` below:
```
return METADATA.getType(mapType(keyType.getTypeSignature(), valueType.getTypeSignature()));
```

this should be as a separate commit

Reply:
Oops, I will fix this.